features/works/work_create.feature

```diff
@@ -197,6 +197,29 @@ Feature: Create Works
       And I should see "Chapter"
       And I should see "1/?"
 
+  Scenario: Creating a new work in a series with some invalid things should not 500
+    Given basic tags
+      And the following activated users exist
+        | login          | password    | email                   |
+        | coauthor       | something   | coauthor@example.org |
+        | badcoauthor    | something   | badcoauthor@example.org |
+      And I am logged in as "thorough" with password "something"
+      And user "badcoauthor" is banned
+    When I set up the draft "Bad Draft"
+      And I fill in "Fandoms" with "Invalid12./"
+      And I fill in "Work Title" with "/"
+      And I fill in "content" with "T"
+      And I check "chapters-options-show"
```

Comment:
In the feature files, we prefer to use the user-facing label text over the field id (in step definitions, the field id is fine) whenever possible.